File: picture/1_load_picture.py
# Load a picture and print its dimensions

import os
import harfang as hg

hg.LoadPlugins()

# mount the system file driver
hg.MountFileDriver(hg.StdFileDriver())

# load picture and display informations
pic = hg.Picture()
ok = hg.LoadPicture(pic, "_data/owl.jpg")
if not ok:
	print("Failed to load image!")
else:
	print("Picture dimensions: %dx%d" % (pic.GetWidth(), pic.GetHeight()))

File: picture/1_load_picture.lua
-- Load a picture and print its dimensions

hg = require("harfang")

hg.LoadPlugins()

-- mount the system file driver
hg.MountFileDriver(hg.StdFileDriver())

-- load picture and display informations
pic = hg.Picture()
ok = hg.LoadPicture(pic, "_data/owl.jpg")
if not ok then
	print("Failed to load image!")
else
	print(string.format("Picture dimensions: %dx%d", pic:GetWidth(), pic:GetHeight()))
end
